src/main/java/me/gaminglounge/gamingteams/listener/AddClearPlaceholder.java

```diff
@@ -0,0 +1,22 @@
+package me.gaminglounge.gamingteams.listener;
+
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerJoinEvent;
+import org.bukkit.event.player.PlayerQuitEvent;
+
+import me.gaminglounge.gamingteams.PlaceholderManager;
+
+public class AddClearPlaceholder implements Listener {
+
+    @EventHandler
+    public void onJoin(PlayerQuitEvent e) {
+        PlaceholderManager.reset(e.getPlayer());
```

Comment:
Is the method name correct?
```suggestion
    @EventHandler
    public void onQuit(PlayerQuitEvent e) {
        PlaceholderManager.reset(e.getPlayer());
```